.github/workflows/coverage.yml

```diff
@@ -115,6 +115,14 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
 
+      fake-gcs-server:
+        image: fsouza/fake-gcs-server:1.47.7
+        ports:
+          - "4443:4443"
+        credentials:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
```

Comment:
Do we need the credentials here? 
I see you just copied the other stuff but I think this is an error on our side too.

Reply:
> Do we need the credentials here?

I do think we don't need credentials to pull service containers. Let me remove them.

Reply:
Removed.